bessie.js

```diff
@@ -0,0 +1,4 @@
+function bessie() {
+	// return "Bessie says hello."
+	return "Bessie says Hello!";
```

Comment:
nit: avoid hard-coding string and use variable `public static final BESSIE_HELLO="Bessie says Hello!"`